opensrp-chw/src/nacp/assets/json.form-sw/pnc_hygiene_observation.json

```diff
@@ -0,0 +1,169 @@
+    },
+    "subscriberid": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "subscriberid",
+      "openmrs_entity_id": "163150AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "simserial": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "simserial",
+      "openmrs_entity_id": "163151AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "phonenumber": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "phonenumber",
+      "openmrs_entity_id": "163152AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "encounter_location": ""
+  },
+  "step1": {
+    "title": "Kuzuia  maambukizi katika jamii",
+    "fields": [
+      {
+        "key": "environment_hygiene_note_one",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "environment_hygiene_note_one",
+        "type": "label",
+        "label_text_style": "italic",
+        "text": "Usafi na uzuiaji wa maambukizi ni pamoja na unawaji mikono unaofaa, usafi wa mwili, utupaji salama wa pedi, kufua nguo zilizochafuliwa, kuzingatia kanuni za usafi unapomshika mtoto mchanga na utunzaji salama wa kitovu."
+      },
+      {
+        "key": "environment_hygiene_spacer",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "environment_hygiene_spacer",
+        "type": "spacer"
+      },
+      {
+        "key": "environment_hygiene_note_two",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "environment_hygiene_note_two",
+        "type": "label",
+        "label_text_style": "italic",
+        "text": "Aidha kinyesi cha mtoto kinaweza kuwa hatari na kinaweza kusababisha magonjwa."
+      },
+      {
+        "key": "defecation_location",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "defecation_location",
+        "type": "edit_text",
+        "hint": "Mtoto wako anajisaidia wapi?"
+      },
+      {
+        "key": "feces_disposal",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "feces_disposal",
+        "type": "edit_text",
+        "hint": "Unatupa kinyesi cha mtoto wako wapi? (Mpe mama nafasi ya kujibu)"
+      },
+      {
+        "key": "is_latrine_present",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "is_latrine_present",
+        "openmrs_data_type": "select one",
+        "type": "spinner",
+        "hint": "Je, una choo kwenye kaya yako?",
+        "values": [
+          "Ndio",
+          "Hapana"
+        ],
+        "keys": [
+          "Yes",
+          "No"
+        ],
+        "openmrs_choice_ids": {
+          "Yes": "1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
The UCS team is  phasing out the use of openmrs uuids, please update this to the same value as key.